edge/key.go

```diff
@@ -29,6 +30,11 @@ func (manager *Manager) SetKey(key string) error {
 		return err
 	}
 
+	u, _ := url.Parse(edgeKey.PortainerInstanceURL)
```

Comment:
We are not checking for an error here. I think we should do.

Reply:
I don't think in this case it's necessary.   An invalid URL will not work and still produce an error.  But the Portainer UI doesn't let you create an endpoint with a bad URL  So it would have to be hand crafted.  In any case, it still won't run properly and produce errors which is expected.